src/main/java/com/itschool/eventmanagment/models/dtos/EventDTO.java

```diff
@@ -41,4 +46,33 @@ public static Event mapEventDtoToEvent(EventDTO eventDTO) {
         event.setMaxParticipants(eventDTO.getMaxParticipants());
         return event;
     }
+
+    public EventDTO(Long id, String name, String location, LocalDateTime eventDateTime, Long userId, int maxParticipants, List<ParticipantDTO> registeredParticipants) {
```

Comment:
Can we rely on some Lombok annotation for this constructor since we have all DTOs properties?